scripts/reliability_metrics.py

```diff
@@ -0,0 +1,263 @@
+#!/usr/bin/env python3.9
+import datetime
+import os
+import logging
+import threading
+import queue
+import re
+import argparse
+from builtins import staticmethod
+
+from ottopia_logging.logging_factory import LoggingFactory
+from ottopia_logging.log_level import LogLevel
+from ottopia_logging.logging_settings import LoggingSettings
+import dataclasses
+
+
+class LoggerSetting(LoggingSettings):
+    """
+    Logger settings
+    """
+
+    DEFAULT_COMPONENT_NAME: str = "reliability-metrics"
+
+    DEFAULT_LOG_LEVEL: LogLevel = LogLevel.INFO
+
+    class Config:
+        """
+        Config for Logger settings
+        """
+
+        arbitrary_types_allowed = True
+
+
+class StateBase:
```

Comment:
do we need this class ?

Reply:
Yes, all other State classes inherit from it, and I use the factory to construct the inherited objects

Reply:
but in line 53 we have another `StateBase`
just move it before `StateFactory`